torch/distributed/fsdp/shard_utils.py

```diff
@@ -188,3 +194,56 @@ def _gather_state_dict(
             tensor = output_tensor
         new_state_dict[key] = tensor
     return new_state_dict
+
+
+def _create_chunk_sharded_tensor(
+    tensor: torch.Tensor,
+    rank: int,
+    world_size: int,
+    device_per_node: int,
+    pg: dist.ProcessGroup,
+) -> ShardedTensor:
+    """
+    Shard a tensor to chunks along the first dimension. The local rank will gets its
+    corresponding chunk as the local shard to create a ShardedTensor.
+    """
+    chunks = tensor.chunk(world_size)
```

Comment:
nit: Do you think it is worth explicitly passing `dim=0` into `chunk()`? It might make it more clear that `chunk()` is chunking along the 0th dimension.